templates/base.html

```diff
@@ -79,80 +79,90 @@
       /* --- */
 
       /* Custom navigation colours */
+      .p-navigation {
+        background-color: #022935;
+      }
       
-        
-        .p-navigation {
-          background-color: #022935;
-        }
-        
 
-        
-        .p-navigation a, .p-navigation a:focus, .p-navigation a:hover, .p-navigation a:visited {
-          color: #fff;
-        }
-        
+      
+      .p-navigation a, .p-navigation a:focus, .p-navigation a:hover, .p-navigation a:visited {
+        color: #fff;
```

Comment:
```suggestion
        color: $color-x-light;
```

Reply:
Actually, this won't work. We don't actually import Vanilla or have any Sass building at all in this project - it's just plain CSS.

And it's not worth the effort of adding this right now, because this site is not long for this work - it will be moved to jaas.ai/docs, and use its templates, CSS, and Sass build pipeline, in the next couple of months.